dspace-api/src/main/java/org/dspace/disseminate/service/CitationDocumentService.java

```diff
@@ -52,15 +48,15 @@ public interface CitationDocumentService {
      * @return true or false
      * @throws SQLException if error
      */
-    public Boolean isCitationEnabledForBitstream(Bitstream bitstream, Context context) throws SQLException;
```

Comment:
Is there a specific reason to change visibility here and canGenerateCitationVersion?

Reply:
@minurmin , taken from the Java Language Specification (JLS)

> If no access modifier is given, the method is implicitly public. 
> It is permitted, but **discouraged** as a matter of style, to redundantly specify the public modifier for a method declaration in an interface.

https://docs.oracle.com/javase/specs/jls/se21/html/jls-9.html#jls-9.4

Reply:
This is a general issue in the DSpace backend code base.